modules/govuk/templates/etc/cron.d/snapshot.erb

```diff
@@ -0,0 +1 @@
+*/10 0-4,6-23 * * * <%= @snapshot_user %> /usr/local/bin/rummager-snapshot
```

Comment:
Any reason not to run this 7 days a week? Are we avoiding 0-6am because of the data sync?

It'd be good to avoid times and use a lock if possible, otherwsie we end up with a lot of timing information that we need to keep in lockstep.